doc/man5/config.pod

```diff
@@ -277,9 +277,25 @@ available to the provider.
 This modules has the name B<alg_section> which points to a section containing
```

Comment:
This modules?

This module reads better.

Reply:
Fixed